### ISSUE TITLE
Greater Than/Less than validator to validate greater than input based on formArray

### ISSUE THREAD

**Issue by Langstra** [rxweb/rxweb#289]
### Description
GreaterThan/LesserThan validator to validate a formArray for successive values. Something like the Unique validator: https://github.com/rxweb/rxweb/tree/master/packages/reactive-form-validators#unique

**Comment by ajayojha** [comment]
@Langstra I have drafted the case of less than, as per my understanding the previous index would be lesser than the next index.

| Index Number | FormControl Value |
| ------------- | ------------- |
| 0  | 10 |
| 1 | 12 |
| 2  | 15 |
| 3 | 17 |

Please correct me If my understanding is wrong. I am planning to release this tonight.

**Comment by Langstra** [comment]
I am sorry for not responding immediately. Yes this would be what I described.

**Comment by ajayojha** [comment]
That’s Ok 🙂, thanks for the response, I will add this feature in next release.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by ajayojha** [comment]
@Langstra The feature is available in the new release of 1.9.9.  Please update the same.
Here is code.

```js
this.productFormGroup = this.formBuilder.group({
            amount:this.formBuilder.array([
              this.getAmountFormGroup()
            ])
        });

     getAmountFormGroup(){
        return this.formBuilder.group({
          amount:['',RxwebValidators.lessThan({isArrayControl:true})]
        })
      }
```
The feature will work in ```lessThan```, ```greaterThan```, ```lessThanEqualTo``` and ```greaterThanEqualTo``` validator.

Please let me know if there is any issue. I am closing this ticket as no further action required on this ticket :).
